routes/things/create.ts

```diff
@@ -2,19 +2,33 @@ import { withRouteSpec } from "lib/middleware/with-winter-spec"
 import { z } from "zod"
 
 export default withRouteSpec({
-  methods: ["POST"],
-  jsonBody: z.object({
+  methods: ["GET", "POST"],
+  urlEncodedFormData: z.object({
     name: z.string(),
     description: z.string(),
   }),
   jsonResponse: z.object({
     ok: z.boolean(),
   }),
 })(async (req, ctx) => {
```

Comment:
sorry bad comment from earlier because you should not have ever changed this file and I was confused. Completely revert this file (there should be no changes to this file)